src/main/resources/static/Test.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Test thingy</title>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/htmx.org/dist/htmx.min.js"></script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/hyperscript.org/dist/_hyperscript.js"></script>
+  <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+  <script>
+    $(document).ready(function() {
+      $("#registrationForm").submit(function(event) {
+        // Prevent the default form submission
+        event.preventDefault();
+
+        // Get form data as an object
+        var formData = {
+          lastname: $("#lastname").val(),
+          firstname: $("#firstname").val(),
+          email: $("#email").val(),
+          password: $("#password").val(),
+          role: $("#role").val()
+        };
+
+        // Log the form data (you can do any manipulation here)
+        console.log(formData);
```

Comment:
remove all console log

Reply:
Test.html was commited by mistake and is now removed